conans/client/profile_loader.py

```diff
@@ -86,7 +86,7 @@ def read_conaninfo_profile(current_path):
 def get_profile_path(profile_name, default_folder, cwd):
     if os.path.isabs(profile_name):
         profile_path = profile_name
-    elif cwd and (os.path.exists(os.path.join(cwd, profile_name)) or profile_name.startswith(".")):
+    elif cwd and profile_name[:2] in ("./", ".\\"):
```

Comment:
My concern is that we are going to break all the profiles including the `default`, it's even documented as `include(default)`. 

http://conanio.readthedocs.io/en/latest/reference/config_files/default_profile.html?highlight=include(default)